setup.py

```diff
@@ -32,7 +32,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.1',
+    version='0.5',
```

Comment:
Thanks, Jeff. If you're ok I'm ok ;)

We could also shoot for exponential versioning, if that's a thing ...